Test/Test_CANBus.c

```diff
@@ -0,0 +1,608 @@
+/* USER CODE BEGIN Header */
```

Comment:
Could this be cleaned up to only include code needed for a test file? Also, include comments at the top describing how to test CAN properly. A similar format can be seen in BPS test files as well.